src/test/scripts/update_versions.py

```diff
@@ -94,13 +94,15 @@ def latest_minor(version, mac_versions):
     logging.info("Product: %s", product)
 
     if product in lts_products:
-        version = product_versions.get_lts_version([product]).replace(suffix, "")
+        version = product_versions.get_lts_version([product]).replace(tag_suffix, "")
         logging.info("Latest LTS version: %s", version)
     else:
         logging.info("Non-LTS product")
         r = requests.get(f'https://marketplace.atlassian.com/rest/2/products/key/{product}/versions/latest')
         version = r.json()['name']
 
-    new_version_tag = f"{version}{suffix}"
-    logging.info(f"Latest version: %s, tagname: {version}{suffix}", version)
+    new_version_tag = f"{version}{tag_suffix}"
+    logging.info(f"Latest version: %s, tagname: {version}{tag_suffix}", version)
     update_versions(product, new_version_tag)
+
+logging.info(">>>> ATTENTION - Don't forget to update the product Changelogs.md - ATTENTION <<<<")
```

Comment:
I always forget this :) We can consider how to automate this one but I don't find this too painful.